public/api/swagger/index.html

```diff
@@ -0,0 +1,65 @@
+<!-- HTML for static distribution bundle build -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Swagger UI</title>
+    <link rel="stylesheet" type="text/css" href="./swagger/swagger-ui.css" />
+    <link
+      rel="icon"
+      type="image/png"
+      href="./favicon-32x32.png"
+      sizes="32x32"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      href="./favicon-16x16.png"
+      sizes="16x16"
+    />
+    <style>
+      html {
+        box-sizing: border-box;
+        overflow: -moz-scrollbars-vertical;
+        overflow-y: scroll;
+      }
+
+      *,
+      *:before,
+      *:after {
+        box-sizing: inherit;
+      }
+
+      body {
+        margin: 0;
+        background: #fafafa;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div id="swagger-ui"></div>
+
+    <script src="./swagger/swagger-ui-bundle.js" charset="UTF-8"></script>
+    <script
+      src="./swagger/swagger-ui-standalone-preset.js"
+      charset="UTF-8"
+    ></script>
+    <script>
+      window.onload = function () {
+        // Begin Swagger UI call region
+        const ui = SwaggerUIBundle({
+          url: "https://raw.githubusercontent.com/calebmech/Book-Bazar/main/documentation/system-design/book-bazar-openapi.yaml",
```

Comment:
It would be nice if this pointed to API documentation for the currently deployed version. Although, this is a minor detail, as I guess the two versions won't get that much out of sync

Reply:
Ah I like this idea, changed it to use `swagger-ui-react` to accomplish this

Reply:
Cool, you should be able to go to login on the preview route, go to `/api/swagger`, try out the `/user` API, and get your user object.

Reply:
Cool! This works great!